src/__tests__/directives.test.ts

```diff
@@ -348,6 +348,123 @@ describe("Execute: handles directives", () => {
         }
       }
     });
+    test("it does not throw [RangeError: invalid String length] or OOM", async () => {
```

Comment:
The test case is testing that include in nested fragments work in general so moving the specific bug case to a line comment instead, as context.
```suggestion
    // Specifically for `[RangeError: invalid String length]` https://github.com/zalando-incubator/graphql-jit/issues/117
    test("it supports use in nested fragments", async () => {
```

Reply:
I would still like to have the tested edge case in the test name, instead of the impact of the bug that we had in the implementation. It will be more helpful to a future developer who sees this test failing.
```suggestion
    test("it supports use in nested fragments", async () => {
```